runtime/compiler/il/J9Symbol.hpp

```diff
@@ -199,7 +199,9 @@ class OMR_EXTENSIBLE Symbol : public OMR::SymbolConnector
 
 private:
 
-   /// This recognized field is currently only used for RecognizedShadows.
+ /**
+  * This recognized field is currently only used for RecognizedShadows.
+  */
```

Comment:
Please indent this comment block by two spaces so that the `/` on the first line lines up with the R in `RecognizedField`